xdem/volume.py

```diff
@@ -0,0 +1,194 @@
+from __future__ import annotations
```

Comment:
I would move Classes to different python files than functions, for readability (otherwise we'll get over 2000 lines for sure). Maybe tdem.py and ddem.py

Reply:
I can move them! @adehecq what do you think of `tdem.py` and `ddem.py`? I'm okay with that. Or maybe a submodule for this? 

```
xdem/submodule/tdem.py
xdem/submodule/ddem.py
```

Reply:
Why not define ddem and tdem in the same file as DEM? So I guess xdem?